### DISCUSSION TITLE
Allow Custom Models and textures

### DISCUSSION THREAD

**Discussion by tobymaxgames** [P3pp3rF1y/SophisticatedBackpacks#1078] [Ideas]
Currently the models for backpacks are hardcoded and cannot be changed by texture packs like [this](https://www.curseforge.com/minecraft/texture-packs/create-sophisticated-backpacks). can this be changed?

**Comment by LopyLuna** [comment]
i agree

**Comment by Lucky-56** [comment]
bump :3

**Comment by Skyphiler** [comment]
figure this out the hard way after a bunch of time making a custom model for cobblemon, the block and item form work but not the outline and entity on your back don't. pls make it customizable

**Comment by checkerddd** [comment]
Big agree.

**Comment by HyperionDash** [comment]
please :3 
   👉👈
